python/templates/selection.xml.jinja2

```diff
@@ -35,17 +42,36 @@
 {{ class_selection(class, postfix='Collection') }}
 {% endfor %}
 
+        <!-- for backwards compatibility with podio versions that didn't
+         generate a version namespace so we generate the "oldschool"
+         un-versioned data here as well -->
```

Comment:
This comment is currently objectively false, as we intend to keep the current version un-namespaced. So, the fact that this is backwards compatible is pretty much by choice.